packages/nextjs/src/server/utils.ts

```diff
@@ -129,11 +128,12 @@ export const nextJsVersionCanOverrideRequestHeaders = () => {
   }
 };
 
-export const injectSSRStateIntoObject = (obj: any, authObject: AuthObject) => {
+export const injectSSRStateIntoObject = <O, T>(obj: O, authObject: T) => {
```

Comment:
❓ shouldn't there be changes to how we call `injectSSRStateIntoObject` (use the added generics) or use default types for `O` and `T` ?
I would expect to find changes like this in this PR:
```
- injectSSRStateIntoObject
+ injectSSRStateIntoObject<any, AuthObject>
```

Reply:
Hey Dimitri, `O` and `T` will be infered and they will automatically take the type of the passed variables